### PR TITLE
Make admin email view more extendable, allow non-User querysets

### PR DESCRIPTION
**Pull request by agjohnson**
This makes the view able to handle non-User objects, for use in messaging users through the admin
on related objects.

### CODE REVIEW

File: readthedocs/templates/core/send_email_form.html [hunk 1]
```diff
@@ -12,14 +12,14 @@
   </p>
 
   <ul>
-    {% for user in users|slice:":20" %}
-      <li>{{ user.email }} ({{ user }})</li>
+    {% for recipient in recipients|slice:":20" %}
```
Comment (ericholscher) [inline readthedocs/templates/core/send_email_form.html]: Is there a reason we're slicing here, instead of in the view?

Comment (ericholscher) [inline readthedocs/templates/core/send_email_form.html]: Nevermind, see below how it's used.

**Comment by ericholscher** [comment]
LGTM other than the comment above.

File: readthedocs/templates/core/send_email_form.html (hunk 1 above)
Comment (agjohnson) [inline readthedocs/templates/core/send_email_form.html]: I'm rather impartial. This isn't important logic, and is for display purposes only, so seems like template logic should be fine.

**Pull request merged**